extensions/amp-auto-ads/0.1/attributes.js

```diff
@@ -29,26 +30,26 @@ const NON_DATA_ATTRIBUTE_WHITELIST = {
 };
 
 /**
- * @param {!Object} configObj
- * @return {!Object<string, string>}
+ * @param {!JsonObject} configObj
+ * @return {!JsonObject<string, string>}
  */
 export function getAttributesFromConfigObj(configObj) {
   if (!configObj['attributes']) {
-    return {};
+    return dict();
   }
   if (!isObject(configObj['attributes']) || isArray(configObj['attributes'])) {
```

Comment:
Is this still enough to qualify for JsonObject? Also, might need to change the warning.